src/Files.Backend/Helpers/NativeFindStorageItemHelper.cs

```diff
@@ -1,12 +1,14 @@
 ﻿// Copyright (c) 2023 Files Community
 // Licensed under the MIT License. See the LICENSE.
 
-using System;
 using System.Runtime.InteropServices;
 using System.Runtime.InteropServices.ComTypes;
 
 namespace Files.Backend.Helpers
 {
+	/// <summary>
+	/// NativeFindStorageItemHelper
+	/// </summary>
 	public class NativeFindStorageItemHelper
```

Comment:
I think you should add something, written like this it provides no information

Reply:
Fixed

Reply:
@lukeblevins I think we should remove this class and introduce Vanara in this project. What do you think?

Following files can be replaced with Vanara like you introduced in your latest PR into Files.App.Storeage.

- NativeFindStorageItemHelper.cs
- NativeFileOperationsHelper.cs
- NativeWinApiHelper.cs
- NativeIoDeviceControlHelper.cs
- NativeDirectoryChangesHelper.cs
- InteropHelpers.cs
- FileUtils.cs

Reply:
@ferrariofilippo What would you think?